src/App.jsx

```diff
@@ -14,30 +16,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [message, setMessage] = useState('No goods selected');
+  const [selectedGood, setSelectedGood] = useState(null);
+  const [selectedIndex, setSelectedIndex] = useState(null);
```

Comment:
you need only one state with a default value `Jam `
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```